javascript/src/main/components/BasicCourseSearch/BasicCourseSearchForm.js

```diff
@@ -36,6 +36,9 @@ const BasicCourseSearchForm = ({ setCourseJSON, fetchJSON }) => {
     const handleSubmit = (event) => {
         event.preventDefault();
         fetchJSON(event, { quarter, subject, level }).then((courseJSON) => {
+            if(courseJSON.total === 0){
+                alert("Empty!");
```

Comment:
using `alert` is not really a good practice here.  The `alert` feature in JavaScript steals focus and annoys end users.

A better way to do this would be by simply displaying a message as part of the component, or using a toast message.

For an example, see: https://github.com/ucsb-cs156-w21/proj-ucsb-courses-search/blob/main/javascript/src/main/pages/Schedule/Schedule.js